### ISSUE TITLE
panic with '--config.label'

### ISSUE THREAD

**Issue by hallyn** [openSUSE/umoci#180]
Not sure whether I'm doing something wrong, but

```bash
ubuntu@ocibuilder:~$ sudo umoci config --image oci:alpine --config.label "serge"
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x63e4a0, 0xc4200100e0)
        /usr/lib/go-1.7/src/runtime/panic.go:500 +0x1a1
main.config(0xc42007a840, 0x0, 0x0)
        /home/ubuntu/go/src/github.com/openSUSE/umoci/cmd/umoci/config.go:257 +0x2990
github.com/openSUSE/umoci/vendor/github.com/urfave/cli.HandleAction(0x633900, 0x69ca40, 0xc42007a840, 0xc42000a700, 0x0)
        /home/ubuntu/go/src/github.com/openSUSE/umoci/vendor/github.com/urfave/cli/app.go:490 +0xd4
github.com/openSUSE/umoci/vendor/github.com/urfave/cli.Command.Run(0x67861b, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x684fba, 0x30, 0x0, ...)
        /home/ubuntu/go/src/github.com/openSUSE/umoci/vendor/github.com/urfave/cli/command.go:210 +0xbb9
github.com/openSUSE/umoci/vendor/github.com/urfave/cli.(*App).Run(0xc4200941a0, 0xc42000a0c0, 0x6, 0x6, 0x0, 0x0)
        /home/ubuntu/go/src/github.com/openSUSE/umoci/vendor/github.com/urfave/cli/app.go:255 +0x7b2
main.main()
        /home/ubuntu/go/src/github.com/openSUSE/umoci/cmd/umoci/main.go:157 +0x753
```

**Comment by cyphar** [comment]
The panic was fixed by #175. Effectively the issue is that labels must be of the form `a=b`.

**Comment by hallyn** [comment]
Gotcha, thanks.
